### PR TITLE
Create profileMapper.py

### PR DESCRIPTION
**Pull request by ZCRNK**

### CODE REVIEW

**Comment by ZCRNK** [comment]
added python framework program

File: README.md [hunk 1]
```diff
@@ -6,30 +6,55 @@ The goal of the framework is to transform the XML dump(s) from EasyDB to the tar
...
+## How To Use:
 
-The Java project can be run in Eclipse for instance.
+* Clone the repository and go into the directory
 
+```bash
+# Clone the repository:
+$ git clone https://github.com/gtadigital/ProfileParser.git
+```
+ A new folder ```ProfileParser```will be created
```
Comment (matteoLorenzini) [inline README.md:22]: Change ProfileParser to ProfileMapper

File: README.md [hunk 2]
```diff
@@ -6,30 +6,55 @@ The goal of the framework is to transform the XML dump(s) from EasyDB to the tar
...
+* Go into the new folder:
 
-* open Java file
-* put path of input XML file between " " in line 10
-* put path of input XSLT file between " " in line 11
-* put path of output XML file between " " in line 16
-* run code
+```bash
+# Go into the repository
+$ cd ProfileParser
```
Comment (matteoLorenzini) [inline README.md:28]: Change ProfileParser to ProfileMapper

File: README.md [hunk 3]
```diff
@@ -6,30 +6,55 @@ The goal of the framework is to transform the XML dump(s) from EasyDB to the tar
...
+# Go into the repository
+$ pip3 install lxml
+```
+   
+* Run the script with arguments:
+
+```bash
+
+# Help documentation
+$ python3 profileParser.py --help
```
Comment (matteoLorenzini) [inline README.md:43]: Change profileParser.py to profileMapper.py

File: README.md [hunk 4]
```diff
@@ -6,30 +6,55 @@ The goal of the framework is to transform the XML dump(s) from EasyDB to the tar
...
+
+OR
+
+# Run the script
+$ python3 profileParser.py --sourcePath "your source path" --targetPath "your target path" --xslt "xslt folder"
```
Comment (matteoLorenzini) [inline README.md:48]: Change profileParser.py to profileMapper.py

File: PythonFrameworkProgram/profileMapper.py [hunk 5]
```diff
@@ -0,0 +1,30 @@
+# author: ETH Zurich, gta digital, Matteo Lorenzini
```
Comment (matteoLorenzini) [inline PythonFrameworkProgram/profileMapper.py:1]: Add Zoe Reinke as author

**Review by matteoLorenzini — CHANGES_REQUESTED** [review]

**Review by matteoLorenzini — APPROVED** [review]

**Pull request merged**